stock_inventory_valuation_location/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Vauxoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+{
+    "name": "Stock Inventory Valuation by Location",
+    "version": "12.0.1.0.0",
+    "author": "Vauxoo, Odoo Community Association (OCA)",
+    "summary": "Introduces an estimation of the value by location in fifo.",
```

Comment:
Does this work only for fifo? From what I understood, it also works for average.

Reply:
Change applied. Thanks